### ISSUE TITLE
About the output size and pretrained model

### ISSUE THREAD

**Issue by KarolyneFarfan** [lemonhdu/Fine-tuned-i3d#1]
Hi, thank you for your work,

I would like to know if I can use extract_features2 to get vectors of 2048 for every window and if it is possible to load the weighs for the kinetics dataset? Also, I understand that I have to get the flow frames and then give them the model, is it right?  If so, could you recommend me a repository or something where I can get the code to apply the flow algorithm ?

Thank you in advance.

**Comment by lemonhdu** [comment]
Hi, KarolyneFarfan. Sorry for the late reply.

To my knowledge, the output dimension of the I3D model should be 1024.  The commonly used 2028-dimension vector should be obtained by concatenating the rgb vector and optical flow vector, both in 1024-dimension.

You can seed the flow frames in a .npy file format into the model. The related two files, i.e., flow_extract.py and save_flowImg_npy.py have been uploaded just now.
